lib/libc/stdio/lib_tmpnam.c

```diff
@@ -99,13 +99,13 @@
 FAR char *tmpnam(FAR char *s)
 {
 	static char path[L_tmpnam];
-	int ret;
+	char *ret;
 
 	if (s == NULL) {
 		s = path;
```

Comment:
Why to declare a new variable path and assign to s?Since path will also be NULL.